tsconfig.json

```diff
@@ -1,20 +1,17 @@
 {
   "compilerOptions": {
     "module": "commonjs",
+    "noImplicitAny": true,
     "outDir": "./build/",
+    "skipLibCheck": true,
     "sourceMap": true,
-    "types": ["node"],
-    "target": "es6",
-    "noImplicitAny": true,
-    "strictNullChecks": true
+    "strictNullChecks": true,
+    "target": "es2017",
+    "typeRoots" : [
```

Comment:
This shouldn't be necessary as I'm pretty sure it's included/used by default

Reply:
Removed `typeRoots` and added `tests/` to the `include` setting. I have confirmed that that the `tests/tests.ts` file is type-checked when `npm test` runs.